style/index.css

```diff
@@ -38,6 +38,30 @@
   height: 100%;
 }
 
+.jp-PDFJSPageNumber {
+  display: flex;
+  align-items: center;
+  padding: 0 6px;
+}
+
+.jp-PDFJSPageNumber input {
+  border: 1px solid var(--jp-border-color0);
+  background-color: var(--jp-input-background);
+  padding: 0 6px;
+  text-align: right;
+  width: 8ch;
```

Comment:
This seemed a little wide to me. I think that 3 characters should be plenty for most PDFs, no? Otherwise we are talking about some seriously long documents.

Reply:
Maybe 8 characters are too many, but 3 are certainly too few. :smile:
In the front matter of a book often there is page "VIII" (4 characters), sometime page "XVIII" (5 characters).
Moreover page label can be an arbitrary string; one frequently used is "Cover" for the cover page.

Reply:
Ah, sure. I had not considered string labels. Makes sense.